vhdx/src/vhdx.rs

```diff
@@ -0,0 +1,73 @@
+mod vhdx_header;
+mod vhdx_metadata;
+
+use crate::vhdx_header::{RegionTableEntry, VhdxHeader};
+
+use std::collections::btree_map::BTreeMap;
+use std::fmt::{self, Display};
+use std::fs::File;
+
+pub enum Error {
+    VhdxHeaderParsingFailed(vhdx_header::Error),
+    VhdxRegionEntryParsingFailed(vhdx_header::Error),
```

Comment:
That's a bit redundant to include `Failed` in an Error name. Add a comment to each error.
```rust
pub enum Error {
    // Failed parsing the VHDX header
    ParseVhdxHeader(vhdx_header::Error),
    // Failed parsing the VHDX region entry
    ParseVhdxRegionEntry(vhdx_header::Error),
}
```

Reply:
Done.